manager.py

```diff
@@ -355,6 +359,23 @@ def handle_cpm(self, this_start, this_end):
         self.data_handler.main(
             self.datalog, cpm, cpm_err, this_start, this_end, counts)
 
+    def extract_git_tag(self):
```

Comment:
I don't think `Manager.handle_git_tag()` is necessary, and `extract_git_tag()` can go in `ServerSender`. See comment there.